packages/react/src/Link/Link.tsx

```diff
@@ -28,15 +40,15 @@ export const Link = forwardRef(
         'amsterdam-link',
         {
           'amsterdam-link--inline': variant === 'inline',
-          'amsterdam-link--standalone-bold': variant === 'standaloneBold',
           'amsterdam-link--in-list': variant === 'inList',
+          'amsterdam-link--in-list__chevron': variant === 'inList' && !icon,
           'amsterdam-link--on-background-light': onBackground === 'light',
           'amsterdam-link--on-background-dark': onBackground === 'dark',
         },
         className,
       )}
     >
-      {variant === 'inList' && <Icon svg={ChevronRight} size="level-7" />}
+      {variant === 'inList' && <Icon svg={icon ? icon : ChevronRight} size="level-7" />}
```

Comment:
Let’s use the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) in cases like this.

```suggestion
      {variant === 'inList' && <Icon svg={icon ?? ChevronRight} size="level-7" />}
```